ui_framework/components/local_nav/_local_search.scss

```diff
@@ -28,6 +28,10 @@
   }
 }
 
+.kuiLocalLuceneSearchInput:focus {
+  padding-right: 13em;
+}
```

Comment:
If someone disables the Kibana plugin then I guess those styles might not be available. If that's a concern, then how about putting them into `ui/public/styles/local_search.less`?

Yep, that uses the correct modifier format. Looks good to me.